### DISCUSSION TITLE
Magic Bell React 10.9.11 not working in NextJS Project

### DISCUSSION THREAD

**Discussion by jmarbutt** [magicbell/community#160] [General]
I am not sure what changed but something is broken in the latest build. The bell is no longer showing and I am seeing errors related to the minification of the files. Not sure how this stopped working.

**Comment by smeijer** [comment]
Hi,

Any chance it's a cache issue? I've just created [a quick sandbox on stackblitz](https://stackblitz.com/edit/nextjs-9jj2rt?file=app%2Finbox.tsx), and things seem to work for me. Also, and maybe more importantly, there aren't any big changes between 10.9.10 and 10.9.11 that are likely to cause build issues (see [the release PR](https://github.com/magicbell-io/magicbell-js/pull/228)).

Can you quote some errors to give me a clue of what's going on?

**Reply by jmarbutt** [reply]
Yeah I am trying to narrow it down. The only errors I am getting are in production with this:
```
 Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings
 ```
 
 I am not sure when this started now actually. It works fine when running locally. I will see if I can adjust your sandbox to cause the issue also.  I do know I have this `experimental` flag on in my next config:
 ```json
   experimental:{
    serverMinification: false
  },
  ```

**Reply by jmarbutt** [reply]
The main thing I am seeing, is it works in local dev but not in the production build.

**Comment by jmarbutt** [comment]
It appears that my build process no longer was picking up the magic bell key, sorry this was my bad.
